Action.podspec

```diff
@@ -20,8 +20,8 @@ Pod::Spec.new do |s|
   s.source_files  = "Sources/**/*.{swift}"
 
   s.frameworks  = "Foundation"
-  s.dependency "RxSwift", "~> 4.3"
-  s.dependency "RxCocoa", "~> 4.3"
+  s.dependency "RxSwift", "~> 4.0"
+  s.dependency "RxCocoa", "~> 4.0"
```

Comment:
Do we have nothing that relays on newer APIs in 4.3 etc ? e.g. Relay, deprecation of Variable, stuff like that?

Reply:
Well I guess if we would, CI wil fail

Reply:
Nop.. we're using the `InputSubject` and it's totally free from those family of `Relay`